src/settings_dialog.js

```diff
@@ -122,7 +122,10 @@ function updateInstalledVersions(){
             "trace:", err.stack);
 
         const li = document.createElement("li");
-        li.textContent = "An error happened: " + err;
+        li.textContent = "Unable to find installed versions in \"" +
+            settings.installPath +
+            "\". Install directory has most likely been moved or deleted. " +
+            "Please select a new install location.";
```

Comment:
I think the wording should be slightly changed, as the error can happen if you just installed the launcher and don't have any versions installed.
So this text should also say that "or you haven't played any Thrive versions yet"